src/main/java/net/shadowmage/ancientwarfare/npc/registry/NpcDefaultsRegistry.java

```diff
@@ -221,9 +221,16 @@ private FactionNpcDefault getSubtypeDefault(String subtype, JsonElement json, Fu
 			npcSubtypeDefault = getLootTable(data).map(npcSubtypeDefault::setLootTable).orElse(npcSubtypeDefault);
 			npcSubtypeDefault = getHeightRange(data).map(npcSubtypeDefault::setHeightRange).orElse(npcSubtypeDefault);
 			npcSubtypeDefault = getThinness(data).map(npcSubtypeDefault::setThinness).orElse(npcSubtypeDefault);
+			if (subtype.contains("spellcaster")) {
```

Comment:
the same in terms of mod compatibility thing being in a core class. Please add a TODO note here.